src/hooks/useGetPasswordFromBiometrics.ts

```diff
@@ -2,12 +2,29 @@ import { useCallback } from 'react';
 import { getBiometricPassword } from 'lib/SecureStorage';
 import { BiometricAuthorizations } from 'types/settings';
 import { useSetAppState } from '@recoil/appState';
+import { Platform } from 'react-native';
+import { appStateOnce } from 'lib/AppStateUtils';
 
 const useGetPasswordFromBiometrics = (biometricAuthorization: BiometricAuthorizations) => {
   const setAppState = useSetAppState();
 
   return useCallback(() => {
     setAppState((currentState) => ({ ...currentState, noSplashScreen: true }));
+    if (Platform.OS === 'android') {
+      // In android if we are requesting the biometrics shortly after the user have
+      // unlocked is phone the application don't show the biometrics popup in this case
+      // we don't need to prevent the visualization of the splash screen on the next
+      // blur event.
```

Comment:
```suggestion
      // On Android if we are requesting the biometrics shortly after the user has
      // unlocked is phone the application won't show the biometrics popup. In this case
      // we don't need to prevent the visualization of the splash screen on the next
      // blur event.
```